modules/serverless-kafka-tls.adoc

```diff
@@ -0,0 +1,46 @@
+// Module is included in the following assemblies:
+//
+// serverless/serverless-kafka.adoc
+
+[id="serverless-configuring-tls-authentication-against-an-apache-kafka_{context}"]
+== Configuring TLS authentication
+
+.Prerequisites
+
+* A Kafka cluster CA certificate as a `.pem` file.
+* A Kafka cluster client certificate and key as `.pem` files.
+
+.Procedure
+
+. Create the certificate files as secrets in your chosen namespace:
++
+----
+$ kubectl create secret --namespace <namespace> generic <kafka-auth-secret> \
+  --from-file=ca.crt=caroot.pem \
+  --from-file=user.crt=certificate.pem \
+  --from-file=user.key=key.pem
+----
++
+[IMPORTANT]
+====
+Use the key names `ca.crt`, `user.crt`, and `user.key`. Do not change them.
+====
+
+. Reference your secret and the namespace of the secret in the `config-kafka` config map:
++
+----
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: config-kafka
+  namespace: knative-eventing
+data:
+  bootstrapServers: <list_of_bootstrap_servers>
+  authSecretName: <kafka_auth_secret>
+  authSecretNamespace: <namespace>
+----
+
+.Additional resources
+
+* link:http://kafka.apache.org/documentation/#security_ssl[TLS on Kafka]
```

Comment:
Link to AMQ Streams doc instead.

Reply:
Also changed the link to https://access.redhat.com/documentation/en-us/red_hat_amq/7.5/html-single/using_amq_streams_on_rhel/index#assembly-kafka-encryption-and-authentication-str